src/main/kotlin/org/rust/lang/core/completion/LookupElements.kt

```diff
@@ -22,113 +22,131 @@ const val KEYWORD_PRIORITY = 10.0
 private const val MACRO_PRIORITY = -0.1
 
 fun createLookupElement(element: RsElement, scopeName: String): LookupElement {
-    val base = LookupElementBuilder.create(element, scopeName)
-        .withIcon(if (element is RsFile) RsIcons.MODULE else element.getIcon(0))
+    val base = element.getLookupElementBuilder(scopeName)
+        .withInsertHandler({ context: InsertionContext, _ ->
+            run {
```

Comment:
It seems that if you drop explicit `()` at `withInsertHandler` call and `run` function, code would be a little bit clearer

Reply:
And maybe extract this lambda as separate function?